backend/workflow_manager/workflow/migrations/0007_remove_workflow_prompt_name_and_more.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.1 on 2024-09-03 08:17
```

Comment:
We don't need migrations in V1

Reply:
Got it, will remove this migration and model change in the v1 version then